jenkins/scripts/integration_clean.sh

```diff
@@ -28,11 +28,22 @@ echo "Cleaning old resources"
 
 for VM_NAME in $VM_LIST
 do
-  # Delete executer vm
+  # Delete executer VM
   echo "Deleting executer VM ${VM_NAME}."
   openstack server delete "${VM_NAME}"
 done
 
+VOLUME_LIST=$(openstack volume list -f json | jq -r '.[] | select(.Name |
+  startswith("ci-test-volume-")) | select((.Name | ltrimstr("ci-test-volume-") |
+  split("-") | .[0] | strptime("%Y%m%d%H%M%S") | mktime) < (now - 21600))
+  | .ID ')
+
+for VOLUME_NAME in $VOLUME_LIST
+do
+  # Delete executer volume
+  echo "Deleting executer volume ${VOLUME_NAME}."
+  openstack volume delete "${VOLUME_NAME}"
```

Comment:
using --force might make sense here, in case the volume is in an unexpected state, maybe ?

Reply:
@maelk Is it only specific to volume or --force is needed also for server,port deletions as well?

Reply:
Well, it is not specific to volumes, but we have seen volumes reach some problematic states,while the others are usually not going into states where it cannot be deleted.

Reply:
Yes, true, added. Thanks!